src/patternfly/upgrade-examples/AlertUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,19 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.alert` | `<div>` | `.pf-c-alert` | `<div>` | Applies default alert styling. Always use with a modifier class. Required. |
+| `.alert-success` | `<div>` | `.pf-m-success` | `.pf-c-alert` | Applies success styling.|
+| `.alert-danger` | `<div>` | `.pf-m-danger` | `.pf-c-alert` | Applies danger styling. |
+| `.alert-warning` | `<div>` | `.pf-m-warning` | `.pf-c-alert` | Applies warning styling. |
+| `.alert-info` | `<div>` | `.pf-m-info` | `.pf-c-alert` | Applies info styling.|
+| `.pficon` | `<div>` | `.pf-c-alert__icon` | `<div>`| Defines the alert icon. Required. |
+| `.alert-link` | `<a>` | `.pf-c-alert-link` | `<a>`| Applies alert link styling. |
+| `--` | `--` | `.pf-c-alert__title` | `<div>` | Defines the alert title. Required. |
+| `--` | `--` | `.pf-c-alert__body` | `<div>` | Defines the alert body. Required. |
+| `--` | `__` | `.pf-c-alert__description` | `<span>`| Defines the alert body. Required. |
```

Comment:
you added this as `.pf-c-alert-description` in the component, although is it necessary? If you're going to keep it, update the "outcome text" so it's more descriptive and remove "Required"

Reply:
going to keep .pf-c-alert-description for now, until we work out if we want the text to be generic or contextual. Just changed this text in the docs